background/main.ts

```diff
@@ -877,30 +907,6 @@ export default class Main extends BaseService<never> {
       }
     )
 
-    transactionConstructionSliceEmitter.on(
-      "signAndSendQuaiTransaction",
```

Comment:
Remove this event from transaction-construction.ts event types as we don't need it(and other places). It is only used to sign and send transactions.